mailman/submit.py

```diff
@@ -61,15 +63,29 @@ def set_status(status):
         sub.save()
         return 0
 
+    # buffer defaults to 7 if value is unset or invalid
+    env_mins = os.getenv('DEADLINE_BUFFER_MINUTES', 7)
+    try:
+        env_mins = int(env_mins)
+    except ValueError:
+        print(
+                f'Invalid DEADLINE_BUFFER_MINUTES value: {env_mins}. '
+                'DEADLINE_BUFFER_MINUTES set to 7.'
```

Comment:
replace 7 with {DEADLINE_BUFFER_MINUTES} (assuming that's what you use above.

Also, print to stderr usingan additional arg to print, e.g. `print(file=sys.stderr, <whatever>)